apps/dashr-financial-report/app.R

```diff
@@ -0,0 +1,1093 @@
+appName <- Sys.getenv("DASH_APP_NAME")
+
+if (appName != ""){
+
+  pathPrefix <- sprintf("/%s/", appName)
+
+
+
+  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+
+             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+
+
+  setwd(sprintf("/app/apps/%s", appName))
+
+}
+
+
+
+#Source assets
+source("assets/FinancialFunctions.R")
+
+# Load Necessary Packages
+library('dash')
+library('dashCoreComponents')
+library('dashHtmlComponents')
+library('plyr')
+library('plotly')
+library('lubridate')
+
+#################################################################################
+
+app <- Dash$new()
+
+
+grey_line <- htmlDiv(list(
+  htmlHr(className = "greyline")
+))
+
+
+overview <- htmlDiv(list(
+  Header()
+))
+
+
+
+#################################################################################
+
+colors <- list(

```

Comment:
It doesn't matter for your app, but using `colors` as a name masks `grDevices::colors`, best to avoid when possible (though honestly it seems like most of the R community doesn't worry about overwriting variables from package to package)

Reply:
Removed it regardless, turns out it was unnecessary lines anyways.